docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,837 @@
+|-------------|---------------------------------------------------------------------------------------------------|--------------------------------|
+| name        | string                                                                                            | 3-32 character command name    |
+| description | string                                                                                            | 1-100 character description    |
+| options?    | array of [ApplicationCommandOption](#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command-option) | the parameters for the command |
+
+## Edit Guild Application Command % PATCH /applications/{application.id#DOCS_TOPICS_OAUTH2/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands/{command.id#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command}
+
+Edit a guild command. Updates for guild commands will be available immediately. Returns `200` and an [ApplicationCommand](#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command) object.
+
+###### JSON Params
+
+| Field       | Type                                                                                              | Description                    |
+|-------------|---------------------------------------------------------------------------------------------------|--------------------------------|
+| name        | string                                                                                            | 3-32 character command name    |
+| description | string                                                                                            | 1-100 character description    |
+| options?    | array of [ApplicationCommandOption](#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command-option) | the parameters for the command |
+
+
+## Delete Guild Application Command % DELETE /applications/{application.id#DOCS_TOPICS_OAUTH2/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/commands/{command.id#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command}
+
+Delete a guild command. Returns `204` on success.
+
+## Create Interaction Response % POST /interactions/{interaction.id#DOCS_INTERACTIONS_SLASH_COMMANDS/interaction}/{interaction.token#DOCS_INTERACTIONS_SLASH_COMMANDS/interaction}/callback
+
+Create a response to an Interaction from the gateway. Functions the same as [Execute Webhook](#DOCS_RESOURCES_WEBHOOK/execute-webhook)
+
+## Edit Original Interaction Response % PATCH /webhooks/application.id/{interaction.token#DOCS_INTERACTIONS_SLASH_COMMANDS/interaction}/messages/@original
+
+Edits the initial Interaction response. Functions the same as [Edit Webhook Message](#DOCS_RESOURCES_WEBHOOK/edit-webhook-messages).
+
+# Delete Original Interaction Response % DELETE /webhooks/application.id/{interaction.token#DOCS_INTERACTIONS_SLASH_COMMANDS/interaction}/messages/@original
```

Comment:
both the delete headers are level 1 instead of level 2 
```suggestion
## Delete Original Interaction Response % DELETE /webhooks/application.id/{interaction.token#DOCS_INTERACTIONS_SLASH_COMMANDS/interaction}/messages/@original
```